pages/build.jsx

```diff
@@ -1,10 +1,13 @@
+import Build from 'components/BuildPage';
 import PageWrapper from 'components/Layout/PageWrapper';
 import Meta from 'components/Meta';
-import Build from 'components/BuildPage';
 
 const BuildPage = () => (
   <PageWrapper>
-    <Meta pageTitle="Build" />
+    <Meta
+      pageTitle="Build"
```

Comment:
there's a small inconsistency in titles, here we have just Build, on other pages we have e.g. `Olas Contribute`, do you think we can remove it if we have Olas in the middle anyway?. it doesn't seem to make any difference when searching (e.g. google search doesn't find contribute page if I search 'olas contribute'), but maybe that's because we don't have a sitemap yet

Reply:
I've gone through and changed the titles to be more consistent with the Build page